app/forms.py

```diff
@@ -0,0 +1,96 @@
+# import re
+from django import forms
+# from PIL import Image
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+# from django.core.exceptions import ValidationError
```

Comment:
Please remove all commented code or state very clearly with a note as to why it should remain here.